src/testing/testing-utils.ts

```diff
@@ -45,18 +45,23 @@ export function expectFiles(fs: d.InMemoryFileSystem, filePaths: string[]): void
   }
 }
 
-export function doNotExpectFiles(fs: d.InMemoryFileSystem, filePaths: string[]) {
-  filePaths.forEach((filePath) => {
-    try {
-      fs.sys.statSync(filePath);
-    } catch (e) {
-      return;
-    }
+/**
+ * Testing utility to validate the non-existence of some provided file paths using a specific file system
+ *
+ * @param fs the file system to use to validate the non-existence of some files
+ * @param filePaths the paths to validate
+ * @throws when one or more of the provided file paths is found
+ */
+export function doNotExpectFiles(fs: d.InMemoryFileSystem, filePaths: string[]): void {
+  const existentFiles: ReadonlyArray<string> = filePaths.filter((filePath: string) => fs.statSync(filePath).exists, []);
```

Comment:
Probably don't need it - IIRC in some cases it can help with the type inference on `filter`, but that may also just be a force of habit from a former life. Fixed in https://github.com/ionic-team/stencil/pull/3435/commits/a1c9e3837545b5b2d68ab71c940ad2f6d22f92ea